spaces/domains.json

```
@@ -218,6 +218,7 @@
   "tribe.toughturtlesnft.com": "toughturtles.eth",
   "vote.rektdao.finance": "daorekt.eth",
   "v.glorylab.xyz": "glorylab.eth",
+  "gov.eazyswap.io":"eazyswap.eth",
```

You will need to add a space between your sub domain and ENS value.
```suggestion
  "gov.eazyswap.io": "eazyswap.eth",
```